pkg/api/login.go

```diff
@@ -176,7 +185,8 @@ func (hs *HTTPServer) trySetEncryptedCookie(ctx *m.ReqContext, cookieName string
 		Value:    hex.EncodeToString(encryptedError),
 		HttpOnly: true,
 		Path:     setting.AppSubUrl + "/",
-		Secure:   hs.Cfg.SecurityHTTPSCookies,
+		Secure:   hs.Cfg.CookieSecure,
```

Comment:
having a helper function on ReqContext to make sure we always those these settings might not be a bad idea. Query to forget.

Reply:
I have a WIP cookie store stashed that we can implement in separate PR.